### PR TITLE
[kf5*] Update to 5.88

### PR DESCRIPTION
**Pull request by wrobelda**
**Describe the pull request**
Updates KF5 frameworks to 5.88

Depends on:
   - [x] #20938 

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  No change

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

**Comment by wrobelda** [comment]
@JackBoosY, @JonLiu1993 this no longer has dependencies and is ready to be merged.

**Comment by wrobelda** [comment]
@JackBoosY did you actually want to request a review from yourself? :)

**Comment by JackBoosY** [comment]
I just noticed myself that don't forgot to review this PR today.

File: ports/ecm/fix_generateqmltypes.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
+diff --git a/modules/ECMGenerateQmlTypes.cmake b/modules/ECMGenerateQmlTypes.cmake
+index d6e124266308028b8533203da63f572f6e99b308..7d7cecb5201521019764102eba0da2abf8b4d911 100644
+--- a/modules/ECMGenerateQmlTypes.cmake
++++ b/modules/ECMGenerateQmlTypes.cmake
+@@ -1,4 +1,5 @@
+ # SPDX-FileCopyrightText: 2017 Aleix Pol Gonzalez <aleixpol@kde.org>
++# SPDX-FileCopyrightText: 2021 Dawid Wróbel <me@dawidwrobel.com>
```
Comment (JackBoosY) [inline ports/ecm/fix_generateqmltypes.patch:7]: May this charset cause some build failures?
We already received many same cases like this.

File: ports/ecm/portfile.cmake [hunk 2]
```diff
@@ -30,3 +31,4 @@ file(INSTALL "${SOURCE_PATH}/COPYING-CMAKE-SCRIPTS" DESTINATION "${CURRENT_PACKA
 # Allow empty include directory
 set(VCPKG_POLICY_EMPTY_INCLUDE_FOLDER enabled)
 
+
```
Comment (JackBoosY) [inline ports/ecm/portfile.cmake:34]: No need to add extra end line here, did you use a script to add this?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/ecm/fix_generateqmltypes.patch (hunk 1 above)
Comment (wrobelda) [reply ports/ecm/fix_generateqmltypes.patch]: Good point, thanks. Fixed.

File: ports/ecm/portfile.cmake (hunk 2 above)
Comment (wrobelda) [reply ports/ecm/portfile.cmake]: Hmm, must have been a mistake somehow. I fixed it now since I was updating the patch anyway.

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the updates!

**Pull request merged**